fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/FHIRPersistenceJDBCCache.java

```diff
@@ -7,6 +7,7 @@
 package com.ibm.fhir.persistence.jdbc;
 
 import com.ibm.fhir.persistence.jdbc.dao.api.ICommonTokenValuesCache;
+import com.ibm.fhir.persistence.jdbc.dao.api.IIdNameCache;
 import com.ibm.fhir.persistence.jdbc.dao.api.INameIdCache;
```

Comment:
I guess its just a reverse of the same info...is it worth trying to combine these?

Reply:
something like http://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/BidiMap.html ...

Reply:
I looked at combining vs. separate. I thought keeping them separate was a little more straight-forward.

Reply:
I'm OK with keeping them separate for now. We're probably going to move to a distribute cache at some point for better scalability, so we look at options then.